tests/Functional/tests/link-span/link-span.html

```diff
@@ -1,3 +1,2 @@
-<p>
-    <a href="http://google.com">Go to <strong>Google</strong></a>
-</p>
+SKIP No subparsing of links
+<p><a href="http://google.com">Go to <strong>Google</strong></a></p>
```

Comment:
As far as I know, this is also not supported by the rst spec. Should we remove this test?